src/styles/index.ts

```diff
@@ -378,6 +378,12 @@ const styles = (theme: ThemeColors) =>
             lineHeight: variables.lineHeightLarge,
         },
 
+        mutedNormalTextLabel: {
+            color: theme.textSupporting,
+            fontSize: variables.fontSizeLabel,
+            lineHeight: variables.lineHeightNormal,
+        },
```

Comment:
Can we just reuse `textLabelSupporting` and add line height style on components that needs it

Reply:
I think we should use generic component for hints with default lineHeight 16px.